src/main/java/com/google/devtools/build/lib/BUILD

```diff
@@ -513,6 +513,7 @@ java_library(
         "//third_party:guava",
         "//third_party:jsr305",
         "//third_party/protobuf:protobuf_java",
+        "//third_party:apache_commons_lang"
```

Comment:
This is a new dependency, and it's totally unnecessary. There is already code in StructImpl.java to escape Java strings into JSON notation (see L319); please re-use it.

Reply:
Sounds good.